packages/core/useAnimate/index.ts

```diff
@@ -0,0 +1,330 @@
+import type { ComputedRef, Ref, ShallowRef, WritableComputedRef } from 'vue-demi'
```

Comment:
import type is best placed at the bottom of all references.

Reply:
Affected by formatting, this effect cannot be achieved. I have looked at other code formats and many places use this.